webpack.config.js

```diff
@@ -1,20 +1,40 @@
+// this lets us require files with JSX/ES6 in them
+require('babel/register');
+
 var path = require('path');
+var React = require('react')
+var HomePage = require('./pages/index.jsx')
+var Sequential = require('./pages/sequential.jsx')
+var ThankYou = require('./pages/thank-you.jsx')
 
-module.exports = {
-  entry: "./client.jsx",
+var getConfig = require('hjs-webpack')
 
+module.exports = getConfig({
+  in: './client.jsx',
+  out: 'public',
+  clearBeforeBuild: false,
   output: {
-    filename: '[name].js',
-    chunkFilename: '[id].chunk.js',
-    path: path.join('public', 'build'),
-    publicPath: '/build/'
+    filename: "main.js",
+    path: path.join(__dirname, 'public')
   },
+  hostname: '0.0.0.0',
+  port: 3000,
+  isDev: process.env.NODE_ENV !== 'production',
+  html: function (data) {
+    var homePageHtmlString = React.createFactory(HomePage)
```

Comment:
We should probably find a better name for this, because homepage is the sequential page, and index.jsx is kinda not a correct name either.

PageWrapper maybe?